src/components/PlayQuizComponent.tsx

```diff
@@ -1,46 +1,78 @@
 import * as React from "react";
-import { Button, Card, Dialog, Pane, Pill, TextInput } from "evergreen-ui";
-import { Quiz } from "../store/model";
+import { Button, Pane, Pill } from "evergreen-ui";
+import { Quiz, Question } from "../store/types";
 import { useState } from "react";
 import { useHistory } from "react-router";
 import { PlayersComponent } from "./PlayersComponent";
+import { ViewQuestionComponent } from "./ViewQuestionComponent";
 
 export const PlayQuizComponent = (props: {
   quiz: Quiz;
-  index: string;
+  qIndex: string;
   editPlayersMode: boolean;
 }) => {
   const quiz = props.quiz;
   const history = useHistory();
 
+  const [questionData, setQuestionData] = useState<{
+    questIndex: number;
+    catIndex: number;
+    question: Question;
+  }>({
+    questIndex: -1,
+    catIndex: -1,
+    question: null,
```

Comment:
This can be better modeled with `[catIndex, questIndex]`. You don't need to set the question because you already have a reference to the quiz. When you pass the question prop to the question component you could do `quiz.categories[questionData.catIndex].questions[questionData.questionIndex]`